scripts/btsbot/_make_tiny_download.py

```diff
@@ -0,0 +1,60 @@
+import argparse
+import numpy as np
+import pandas as pd
+import os
+import shutil
+
```

Comment:
Write a docstring explaining what this file is used for and linking to the tiny data that was created with it.